pkg/service/manifest/service.go

```diff
@@ -157,7 +160,24 @@ func (s Service) CreateManifest(ctx context.Context, request model.CreateManifes
 		)
 	}
 	if request.PresentationDefinition != nil {
```

Comment:
should this be called PresentationDefinitionRef?

Reply:
Good call, done.